packages/flutter/lib/src/widgets/binding.dart

```diff
@@ -148,6 +148,53 @@ abstract class WidgetsBindingObserver {
   ///    boilerplate.
   void didChangeMetrics() { }
 
+  /// Called when the platform's text scale factor changes.
+  ///
+  /// This typically happens as the result of the user changing system
+  /// preferences, and it should affect all of the text sizes in the
+  /// application.
+  ///
+  /// ```dart
```

Comment:
this should definitely be marked `## Sample code`, to ensure it's properly analyzed

Reply:
Cool, I didn't know about the "## Sample code" thing.  Done.